### PR TITLE
Document missing game parameters

### PR DESCRIPTION
**Pull request by abant07**
I am not sure if I did the documentation correctly for all methods, so if you find any mistakes, please point them out.

Thanks

### CODE REVIEW

**Review by Segelzwerg — CHANGES_REQUESTED** [review]
@abant07 thanks for your PR. I think there is something wrong with indentation. We use 4 spaces in our projects and seems you used 2.

Also thanks for writing documentation for the private methods which we still can keep (haven't checked them though), but the task was to write the documentation of the parameters of the route method.

**Comment by iTitus** [comment]
Just a quick question @abant07, are you using the web ui to edit the files?
Have you ever run the application or the tests locally?

**Comment by abant07** [comment]
Hi @iTitus 

Right now, I have been editing the on the GitHub website because it is easier, but I have GitHub downloaded on my desktop, so I could run locally.

**Comment by abant07** [comment]
Hi @Segelzwerg 

I have just finished with the documentation for this issue. Are there any other changes I should make?

Thanks

**Comment by Segelzwerg** [comment]
> Hi @Segelzwerg 
> 
> I have just finished with the documentation for this issue. Are there any other changes I should make?
> 
> Thanks

@abant07 as I mentioned above the documentation is required in the route method. The privates are optional. We can keep them, previously we didn't bother writing them.

File: whist/server/api/game/create.py [hunk 1]
```diff
@@ -29,6 +29,13 @@ def create_game(request: Dict[str, str], user: Player = Depends(get_current_user
 
 
 def _set_game_parameter(request, user, pwd_service: PasswordService):
+    """
+    Sets the maximum and minimum level for players to join game.
```
Comment (Segelzwerg) [inline whist/server/api/game/create.py:33]: It sets all game parameters.

File: whist/server/api/game/create.py [hunk 2]
```diff
@@ -29,6 +29,13 @@ def create_game(request: Dict[str, str], user: Player = Depends(get_current_user
...
+    :param request: Must contain a 'game_name'. 'password' is optional
+    :param user: that created the game session.
+    :param pwd_service: service to handle password requests.
+    :return: the players in the game
```
Comment (Segelzwerg) [inline whist/server/api/game/create.py:37]: No, please have a look again

File: whist/server/api/game/create.py [hunk 3]
```diff
@@ -44,6 +51,11 @@ def _set_game_parameter(request, user, pwd_service: PasswordService):
 
 
 def _get_amount_player(request, key) -> Optional[int]:
+    """
+    Checks to see if player is in the level range.
+    :param request: Must contain a 'game_name'. 'password' is optional
+    :param key: the player entering game
+    """
```
Comment (Segelzwerg) [inline whist/server/api/game/create.py:58]: Try to understand what this method actually does. We don't even have player level, but ratings.

File: whist/server/api/game/create.py [hunk 4]
```diff
@@ -52,6 +64,11 @@ def _get_amount_player(request, key) -> Optional[int]:
 
 
 def _get_game_name(request):
+    """
+    Creates a game name.
```
Comment (Segelzwerg) [inline whist/server/api/game/create.py:68]: Wording: the name is not created here

File: whist/server/api/game/create.py [hunk 5]
```diff
@@ -52,6 +64,11 @@ def _get_amount_player(request, key) -> Optional[int]:
...
+    :param request: Must contain a 'game_name'. 'password' is optional
```
Comment (Segelzwerg) [inline whist/server/api/game/create.py:69]: Password not relevant

File: whist/server/api/game/create.py [hunk 6]
```diff
@@ -60,6 +77,12 @@ def _get_game_name(request):
 
 
 def _get_password(pwd_service, request):
+    """
+    Creates a password for game.
+    :param request: Must contain a 'game_name'. 'password' is optional
+    :param pwd_service: service to handle password requests.
+    :return: the password for game instance
+    """
```
Comment (Segelzwerg) [inline whist/server/api/game/create.py:85]: See above

**Review by Segelzwerg — CHANGES_REQUESTED** [review]

**Comment by iTitus** [comment]
Hi @abant07 
> I have GitHub downloaded on my desktop, so I could run locally.

Please do that, if you use VSCode or an IDE like PyCharm to edit the files everything will be much easier for you, and for us as well!

**Comment by Segelzwerg** [comment]
@abant07 thanks for writing the documentation for the private method, but the requirement is to expand the parameter doc in L19.

**Review by Segelzwerg — CHANGES_REQUESTED** [review]
@abant07 thanks for writing the documentation for the private method, but the requirement is to expand the parameter doc in L19.

**Pull request closed without merging**